openhands/runtime/utils/edit.py

```diff
@@ -138,15 +145,17 @@ def __init__(self, enable_llm_editor: bool, *args: Any, **kwargs: Any) -> None:
         draft_editor_config = self.config.get_llm_config('draft_editor')
 
         # manually set the model name for the draft editor LLM to distinguish token costs
-        llm_metrics = Metrics(model_name='draft_editor:' + draft_editor_config.model)
+        Metrics(model_name='draft_editor:' + draft_editor_config.model)
```

Comment:
this is not assigned to any variable, is this intentional?

Reply:
```suggestion
```